scripts/missions/windurst/2_1_Lost_for_Words.lua

```diff
@@ -0,0 +1,364 @@
+-----------------------------------
+-- Lost for Words
+-- Windurst M2-1
+-----------------------------------
+-- !addmission 2 3
+-- Rakoh Buuma       : !pos 106 -5 -23 241
+-- Mokyokyo          : !pos -55 -8 227 238
+-- Janshura-Rashura  : !pos -227 -8 184 240
+-- Zokima-Rokima     : !pos 0 -16 124 239
+-- Tosuka-Porika     : !pos -26 -6 103 238
+-- Nanaa Mihgo       : !pos 62 -4 240 241
+-- Fossil Rocks      : !pos 17 18 184 198
+-- Mahogany Door     : !pos -11 0 20 192
+-- House of the Hero : !pos -26 -13 260 239
+-----------------------------------
+require('scripts/settings/main')
+require('scripts/globals/items')
+require('scripts/globals/missions')
+require('scripts/globals/npc_util')
+require('scripts/globals/keyitems')
+require('scripts/globals/interaction/mission')
+require('scripts/globals/zone')
+-----------------------------------
+local mazeID = require("scripts/zones/Maze_of_Shakhrami/IDs")
+-----------------------------------
+
+local mission = Mission:new(xi.mission.log_id.WINDURST, xi.mission.id.windurst.LOST_FOR_WORDS)
+
+mission.reward =
+{
+    rankPoints = 300,
+}
+
+local handleAcceptMission = function(player, csid, option, npc)
+    if option == 3 then
+        mission:begin(player)
+        player:setMissionStatus(mission.areaId, 1)
+        player:messageSpecial(zones[player:getZoneID()].text.YOU_ACCEPT_THE_MISSION)
+    end
+end
+
+local examineRock = function(player, npc)
+    local rockOffset = npc:getID() - mazeID.npc.FOSSIL_ROCK_OFFSET
+    local correctRock = mission:getVar(player, "Rock")
+    local missionStatus = player:getMissionStatus(mission.areaId)
+
+    if rockOffset == correctRock then
+        if missionStatus == 3 then
+            player:setMissionStatus(mission.areaId, 4)
+            return mission:keyItem(xi.ki.LAPIS_CORAL)
+        elseif missionStatus == 4 then
+            return mission:messageSpecial(mazeID.text.NO_NEED_INVESTIGATE)
+        end
+    else
+        -- TODO: This multiline message should be converted to messageName, however the equivalent in
```

Comment:
Before this PR is merged, can you turn any outstanding TODOS into issues that contain links to the offending lines so we can track them better